web/html/home.html

```diff
@@ -84,6 +84,8 @@ <h2>#3 Autoplay</h2>
         <button data-select="play" data-id="slider-3" class="btn green disabled">Play</button>
         <button data-select="prev" data-id="slider-3" class="btn">Previous</button>
         <button data-select="next" data-id="slider-3" class="btn">Next</button>
+        <button data-select="remove-poh" data-id="slider-3" class="btn">Remove PauseOnHover</button>
+        <button data-select="register-poh" data-id="slider-3" class="btn">Add PauseOnHover</button>
```

Comment:
There should be only one clickable button at a time. You can either use `.disabled` class (see Pause and Play button) or display none/block as we also don't have so much spacing here to have so many buttons.